### ISSUE TITLE
mds_services in model not in sync with reality

### ISSUE THREAD

**Issue by saelbrec** [openvstorage/framework#1395]
Found this on GiG's be-g8-4 environment(Fargo RC2)

```
In [12]: for vd in vdisks:
    if len(vd.mds_services) != 3:
        print len(vd.mds_services), len(vd.info['metadata_backend_config'])
   ....:         
6 3
4 3
5 3
6 3
6 3
5 3
4 3
6 3
6 3
01
```

```
In [15]: for serv in vd.mds_services:
   ....:     print serv
   ....:     
{
    "is_master": false, 
    "mds_service_guid": "4076e5e6-554d-49c4-bc63-5fad7131d98d", 
    "guid": "00987667-9f5e-406a-a810-977be360eab6", 
    "vdisk_guid": "fe195b10-7f70-4f75-bd63-3ecf3ba2fc65"
}
{
    "is_master": false, 
    "mds_service_guid": "4076e5e6-554d-49c4-bc63-5fad7131d98d", 
    "guid": "316bdd83-cd73-4d44-a280-967146932bd8", 
    "vdisk_guid": "fe195b10-7f70-4f75-bd63-3ecf3ba2fc65"
}
{
    "is_master": false, 
    "mds_service_guid": "bcbde070-8f52-4ce0-a37c-89762a14c84c", 
    "guid": "36a2a7cf-7139-4540-b9b7-234c7cd809e9", 
    "vdisk_guid": "fe195b10-7f70-4f75-bd63-3ecf3ba2fc65"
}
{
    "is_master": true, 
    "mds_service_guid": "436dd7cf-73e1-452d-a7be-bf9533336704", 
    "guid": "7cff4a16-34b0-4399-b9d3-fd17f5f364b3", 
    "vdisk_guid": "fe195b10-7f70-4f75-bd63-3ecf3ba2fc65"
}
{
    "is_master": true, 
    "mds_service_guid": "436dd7cf-73e1-452d-a7be-bf9533336704", 
    "guid": "916620ad-2116-48c2-8250-f0b86680552e", 
    "vdisk_guid": "fe195b10-7f70-4f75-bd63-3ecf3ba2fc65"
}
```

**Comment by khenderick** [comment]
Only an issue in the model, shouldn't have any real impact since e.g. reconfigurations always start from reality.

**Comment by khenderick** [comment]
Fixed by #1420, packaged on dev in openvstorage-2.7.12-dev.1486469129.a537168

**Comment by khenderick** [comment]
openvstorage-2.7.12.1
